images/csi-driver-smb.yml

```
@@ -0,0 +1,29 @@
+content:
+  source:
+    dockerfile: Dockerfile.openshift
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/csi-driver-smb.git
+      web: https://github.com/openshift/csi-driver-smb
+    ci_alignment:
+      streams_prs:
+        ci_build_root:
+          stream: rhel-9-golang-ci-build-root
+        auto_label:
+        - approved
+        - lgtm
```

```suggestion
      streams_prs:
        commit_prefix: "UPSTREAM: <carry>: "
        ci_build_root:
          stream: rhel-9-golang-ci-build-root
        auto_label:
        - approved
        - lgtm
```
add the commit_prefix so ART bot PR's don't fail our verify-commits job